src/recode.jl

```diff
@@ -11,6 +11,8 @@ or [`in`](@ref) the key (first item of the pair), then the corresponding value
 (second item) is copied to `dest`.
```

Comment:
On the previous line, mentioning `in` is no longer correct. Maybe say something like ``if the element is equal to (according to [`isequal`](@ref))) the key (first item of the pair) or to one of its entries if it is a collection``.

Reply:
fixed